composer.json

```diff
@@ -40,7 +40,8 @@
 		"pear-pear.horde.org/horde_util": "^2.5.8@stable",
 		"pear-pear.horde.org/horde_smtp": "^1.9.5@stable",
 		"psr/log": "^1",
-		"rubix/ml": "0.2.4"
+		"rubix/ml": "0.2.4",
+		"pear-pear.horde.org/horde_stringprep": "^1.0"
```

Comment:
bonus points for moving the line up in alphabetical order :P

Reply:
Added the sort-packages flag. Next time composer will do for us ;)